src/rest-server/src/models/v2/job/k8s.js

```diff
@@ -288,6 +301,26 @@ const convertFrameworkDetail = async (framework) => {
       ),
     };
   }
+
+  if (launcherConfig.enabledHived) {
+    const affinityGroups = {};
+    (await axios({
+      method: 'get',
```

Comment:
If this get http call failed, log it, and set lazyPreemption to unknown, instead of lazyPreemption=false or =true.

Reply:
Currently, null lazyPreemptionStatus has 2 meaning, one is lazyPreemption=false, another is lazyPreemption=unknown. I meant that, you need to differenate it, so you may need to introduce another field to task status.

---
In reply to: [356571019](https://github.com/microsoft/pai/pull/4007#discussion_r356571019) [](ancestors = 356571019)